src/client/opamArg.ml

```diff
@@ -915,10 +920,17 @@ let build_options =
        actually performing them. \
        WARNING: This option is dangerous and likely to break your opam \
        environment. You probably want `--dry-run'. You've been warned." in
+  let ignore_constraints_on =
+    mk_opt ~section ["ignore-constraints-on"] "PACKAGES"
+      "Forces opam to ignore version constraints on all dependencies to the \
+       listed packages. This can be used to actually test compatibility, but \
+       expect builds to break when using this. Note that version constraints \
+       on optional dependencies and conflicts are unaffected."
```

Comment:
> As for removing version constraints on depexts, I am not sure this would be expected, or what purpose it would serve..

If I have `A` that has a constrainted depext on `B` and want to install `A` without the constraint on `B` no ?